lib/auth/auth.go

```diff
@@ -3997,6 +4008,28 @@ func (a *Server) ExtendWebSession(ctx context.Context, req WebSessionReq, identi
 	return sess, nil
 }
 
+func decodeDeviceExtensionsFromSession(webSession types.WebSession) (*tlsca.DeviceExtensions, error) {
+	// Reading the extensions from the session itself means we are always taking
+	// them for a legitimate source (ie, certificates issued by Auth).
```

Comment:
Therefore we needn't validate the cert here? Consider clarifying

Reply:
Correct, I'll clarify.